onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -1536,5 +1542,57 @@ TEST(InferenceSessionTests, TestParallelExecutionWithCudaProvider) {
 
 #endif
 
+TEST(InferenceSessionTests, ModelThatTriggersAllocationPlannerBug) {
+  SessionOptions so;
```

Comment:
Would be great to have a comment saying what the bug was or give the test a more meaningful name. Not clear from the model name that it's about not re-using a buffer for a string that was not previously a string.

Reply:
done